README.md

```diff
@@ -1,13 +1,75 @@
 # Flocking MIDI Router
 
-A very simple MIDI router, made with Flocking.
+A very simple Node-based MIDI router, made with [Flocking](http://flockingjs.org/).  Routes all messages from one or
+more inputs to one or more outputs.
+
+## Installation
+
+Before you can use this package, you will need to install its dependencies using a command like `npm install` or
+`yarn install`.
 
 ## Usage
 
+The most basic way to launch the route is using a command like:
+
+`node launcher.js`
+
+You can also install flocking-midi-router as a global command available in your path by running `npm link` (or `yarn
```

Comment:
Yes, those instructions are to workaround the lack of a published package, and yes, IMO you should at least fluid-publish a dev release.